packages/data/.gitignore

```diff
@@ -15,15 +15,11 @@
 *.iws
 .idea/
 
-# The .vscode folder contains launch configuration and tasks you configure in
-# VS Code which you may wish to be included in version control, so this line
-# is commented out by default.
-#.vscode/
-
 # Flutter/Dart/Pub related
 # Libraries should not include pubspec.lock, per https://dart.dev/guides/libraries/private-files#pubspeclock.
 /pubspec.lock
 **/doc/api/
 .dart_tool/
 .packages
 build/
+/lib/di/injector.config.dart
```

Comment:
try to commonize all injector.config.dart files in root gitignore, like **.injector.config.dart. or smth like this